features/beepBoop.js

```diff
@@ -1,25 +1,20 @@
-const { directMention } = require("@slack/bolt");
+// const { diretMention } = require("@slack/bolt");
```

Comment:
Typo got introducted in this function name. Not technically an issue, but will probably confuse someone if it's uncommented.

Reply:
good catch, it's fixed